MoreLinq/NestedLoops.cs

```diff
@@ -1,6 +1,6 @@
 #region License and Terms
 // MoreLINQ - Extensions to LINQ to Objects
-// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
+// Copyright (c) 2018 Leandro F. Vieira (leandromoh). All rights reserved.
```

Comment:
You cannot change copyright notice of the “original work” so please revert this line.